src/main/java/windycall/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package windycall;
+
+public abstract class OperationHandler {
+
+    public abstract void handle(String userCommand);
```

Comment:
I see there are static handle functions and an instance handle functions. Maybe consider renaming or removing one? I don't believe this (static) is used. I believe you may have tried to override this static function. I also tried that and realised it was not possible.

Reply:
Yes, I do have this issue when coding. So far I just use static to make it functionable, and I think no need to do the renaming because I can overload this handle methods specifically in different classes while maintaining ```public void handle(String)```.
Anyways, thanks for sharing your own experience, I'll have a try!!